cms/standard_pages/models.py

```diff
@@ -38,3 +40,76 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class IndexPage(BasePage):  # type: ignore[django-manager-missing]
+    template = "templates/pages/index_page.html"
+
+    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "IndexPage", "InformationPage"]
+    subpage_types: ClassVar[list[str]] = ["IndexPage", "InformationPage"]
+
+    description = models.TextField(blank=True)
+    featured_pages = StreamField([("featured_page", RelatedContentBlock())], blank=True)
```

Comment:
given we allow both pages and external links, this is not really `featured_pages`, no? The dev notes indicate calling this `featured_items`.

Reply:
additionally, this could use some `help_text` that indicates that populating this field will stop the index from automatically showing a list of child pages"

Reply:
Done in

Reply:
Done in [First round of renaming and refactoring (updated the migration file)](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/19a565709a37e4d47e83b51f46f0af0de718413a)